internal/db/schema/migrations/oss/postgres/88/01_remove_session_connection_state.up.sql

```diff
@@ -0,0 +1,221 @@
+  end;
+  $$ language plpgsql;
+
+  create trigger update_session_state_on_termination_reason after update of termination_reason on session
+    for each row execute procedure update_session_state_on_termination_reason();
+
+  create function wh_insert_session_connection() returns trigger
+    as $$
+    declare
+  new_row wh_session_connection_accumulating_fact%rowtype;
+  begin
+    with
+      authorized_timestamp (date_dim_key, time_dim_key, ts) as (
+        select wh_date_key(create_time), wh_time_key(create_time), create_time
+          from session_connection
+         where public_id = new.public_id
+           and connected_time_range is null
+      ),
+      session_dimension (host_dim_key, user_dim_key, credential_group_dim_key) as (
+        select host_key, user_key, credential_group_key
+          from wh_session_accumulating_fact
+         where session_id = new.session_id
+      )
+    insert into wh_session_connection_accumulating_fact (
+                connection_id,
+                session_id,
+                host_key,
+                user_key,
+                credential_group_key,
+                connection_authorized_date_key,
+                connection_authorized_time_key,
+                connection_authorized_time,
+                client_tcp_address,
+                client_tcp_port_number,
+                endpoint_tcp_address,
+                endpoint_tcp_port_number,
+                bytes_up,
+                bytes_down
+    )
+    select new.public_id,
+           new.session_id,
+           session_dimension.host_dim_key,
+           session_dimension.user_dim_key,
+           session_dimension.credential_group_dim_key,
+           authorized_timestamp.date_dim_key,
+           authorized_timestamp.time_dim_key,
+           authorized_timestamp.ts,
+           new.client_tcp_address,
+           new.client_tcp_port,
+           new.endpoint_tcp_address,
+           new.endpoint_tcp_port,
+           new.bytes_up,
+           new.bytes_down
+      from authorized_timestamp,
+           session_dimension
+ returning * into strict new_row;
+    return null;
+  end;
+  $$ language plpgsql;
+
+  create trigger wh_insert_session_connection after insert on session_connection
+    for each row execute function wh_insert_session_connection();
+
+  create function wh_insert_session_connection_state() returns trigger
+  as $$
+    declare
+               state text;
+            date_col text;
+            time_col text;
+              ts_col text;
+                   q text;
+      connection_row wh_session_connection_accumulating_fact%rowtype;
+    begin
+      if new.connected_time_range is null then
+        -- Indicates authorized connection. The update statement in this
+        -- trigger will fail for the authorized state because the row for the
+        -- session connection has not yet been inserted into the
+        -- wh_session_connection_accumulating_fact table.
+        return null;
+      end if;
```

Comment:
This new condition is why the [wh/session_connection/update](https://github.com/hashicorp/boundary/blob/irindos-session-connection-states/internal/db/sqltest/tests/wh/session_connection/update.sql) sqltest is failing. I think that test just needs to be updated so it is setting the new `connection_time_range` column when it updates the session_connection.

Reply:
ty!